openetl_utils/connector_utils.py

```
@@ -10,7 +10,11 @@
 home = os.environ['OPENETL_HOME']
 sys.path.append(home)
 import json
-from openetl_utils.database_utils import DatabaseUtils
+try:
+    from openetl_utils.database_utils import DatabaseUtils
+except  ImportError as e:
```

Ruff (0.14.1)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)